schedule/yast/modify_existing_partition_sle.yaml

```diff
@@ -1,7 +1,8 @@
 ---
-name:           modify_existing_partition
+name:           modify_existing_partition@aarch
 description:    >
   Installation where we modify some pre-existing partitions. Must depend on some create_hdd test suite.
+!include: test_data/yast/modify_existing_partition.yaml
```

Comment:
same here, `!include` without `test_data`.

Reply:
fixed

Reply:
what do you mean without test_data